compiler/rustc_ast/src/ast.rs

```diff
@@ -2537,11 +2537,20 @@ pub enum UseTreeKind {
     /// `use prefix` or `use prefix as rename`
     Simple(Option<Ident>),
     /// `use prefix::{...}`
-    Nested(ThinVec<(UseTree, NodeId)>),
+    Nested(UseTreeNested),
```

Comment:
Why not just add a `Span` member here? Then you don't need to do all those `nested` -> `nested.items` changes below, and you can just ignore the span where it's not used with `_`.

Reply:
Yes, I think that would be much better.